src/tests/system/tests/test_sssctl.py

```diff
@@ -1121,9 +1122,7 @@ def test_sssctl__analyze_child_logs(client: Client, ldap: LDAP):
     with pytest.raises(SSHAuthenticationError):
         client.ssh("user1", "Wrong").connect()
     result = client.sssctl.analyze_request("show --pam --child 1")
-    assert (
-        "Authentication failure to the client" in result.stdout
-    ), "'Authentication failure to the client' was not found"
+    assert "Preauthentication failed" in result.stdout, "'Preauthentication failed' was not found"
```

Comment:
Nitpick, can you add a ! at the end of the error message.

Reply:
Done.